common/src/main/java/io/netty/util/internal/logging/Slf4JLoggerFactory.java

```diff
@@ -40,31 +37,8 @@ public Slf4JLoggerFactory() {
 
     Slf4JLoggerFactory(boolean failIfNOP) {
         assert failIfNOP; // Should be always called with true.
-
-        // SFL4J writes it error messages to System.err. Capture them so that the user does not see such a message on
```

Comment:
Does this provide any utility or was it was a mistake to do this in the first place? The original issue seems to focus more on the content is not flushed at the appropriate time. I wonder if the original intention was to capture the output and then flush it after the logger is setup ... not sure though.